virt/virt-2-6-release-notes.adoc

```diff
@@ -65,6 +65,15 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 === Web console
 
 //CNV-9312: The VM wizard has been completely redesigned, simplifying VM creation
+The *Create Virtual Machine Template* wizard enables you to:
+
+* Select a preconfigured Red Hat template, customize, review, and save it as a custom virtual machine template. Preconfigured Red Hat templates listed in the *Templates* tab are available for different versions of Red Hat Enterprise Linux, Fedora, Microsoft Windows 10, and Microsoft Windows Servers.
+
+* Create a virtual machine template by completing the fields for the *General* step in the wizard, reviewing the settings, and saving the custom template.
+
+The availability of preconfigured templates expedites the process for creating virtual machines.
+
+Use the *Create Virtual Machine from Template* wizard to select a Red Hat template or a custom template, customize, review and create a virtual machine. If the selected template does not have a boot source, you are prompted to add a boot source.
```

Comment:
The similarity of this point to the earlier point makes it seem like a rogue copy-paste. This also seems like too much info for a release note. I think this and the previous para can be simplified to something like:
The virtual machine wizard has been redesigned to simplify the process of creating virtual machines by using virtual machine templates.
With an xref to the relevant content

Reply:
See the tight write-up now with the xrefs. Life is good!